dap-python.el

```diff
@@ -233,8 +234,16 @@ strings, for the sake of launch.json feature parity."
        (unless (plist-get conf :cwd)
          (cl-remf conf :cwd))
 
-       (plist-put conf :dap-server-path
-                  (list python-executable "-m" "debugpy.adapter")))
+       (pcase (plist-get conf :request)
+         ("launch"
+          (plist-put conf :dap-server-path
+                     (list python-executable "-m" "debugpy.adapter")))
+         ("attach"
+          ;; TODO: Replace the hardcoded values below with the sub-values of
+          ;; :connect
+          (plist-put conf :host "127.0.0.1")
```

Comment:
I think this can be done in this PR, as 127.0.0.1 and 5678 might not be the only attach ports.

Reply:
I just pushed an update to address this.